### DISCUSSION TITLE
option to show all annotations in a dashboard

### DISCUSSION THREAD

**Discussion by kevingessner** [grafana/grafana#38178] [Dashboards, Panels & Visualizations]
**What would you like to be added**: When setting up Grafana-backed annotations on a dashboard, add an option to show all annotations, regardless of tag or original dashboard.

**Why is this needed**: Currently, the options for showing Grafana-backed annotations are "Dashboard" and "Tags". Dashboard mode shows only the annotations created in each panel; Tags mode shows only those with a specific tag.  There's no way to view all annotations, which is useful in some circumstances (such as high-level overview dashboards or in small systems where tagging is overkill).

I've submitted #29033 to show all annotations (instead of none of them) when Tags mode is selected with no tags. That avoids adding a new option but makes a currently-useless configuration do something useful. (If you do want to show no annotations, just turn them off instead).

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I believe this is implemented under the dashboard settings > Annotations. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if this is not a solution and you would like to request the feature.

**Comment by matzl** [comment]
For future readers stumbling across this post, here is how you list annotations in Grafana:
https://stackoverflow.com/a/78053869/2738240

**Reply by gelicia** [reply]
Thank you so much!
